src/app/calculator/compressed-air/compressed-air-pressure-reduction/compressed-air-pressure-reduction.service.ts

```diff
@@ -135,11 +146,22 @@ export class CompressedAirPressureReductionService {
   generateExample(settings: Settings, isBaseline: boolean): CompressedAirPressureReductionData {
     let proposedPressure: number = 90;
     let pressure: number = 100;
+
+    //TODO: DEFAULTS
```

Comment:
There are a couple TODOs here and in mockTreasurehunt

Reply:
Good catch!